### DISCUSSION TITLE
After upgrading from MariaDB to PostgreSQL: "APPS_NOT_READY_WARNING"

### DISCUSSION THREAD

**Discussion by saschafoerster** [pretalx/pretalx#1881] [Self-hosting]
I get this error message: 

```
/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py:98: RuntimeWarning: Accessing the database during app initialization is discouraged. To fix this warning, avoid executing queries in AppConfig.ready() or when your app modules are imported.
  warnings.warn(self.APPS_NOT_READY_WARNING_MSG, category=RuntimeWarning)
```
I tried to remove all plugins, but the error message stays. 

I am using the docker-container pretalx/standalone, that is prepared. Any hint or help would be very welcome, as this error message is flooding (and appears every time my cronjobs run). :)

**Comment by rixx** [comment]
This is not an error message but a warning message. While it’s true that it’s not great for it to show up, please note that it’s not breaking anything and that tasks still execute as per usual. Further discussion in #1861.
